package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "format-error",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Can you undo this change? I prefer to bump this in an explicit commit before publish.